index.html

```diff
@@ -3,31 +3,71 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Quiz App</title>
+    <title>Quiz-App</title>
     <link rel="stylesheet" href="styles/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+      integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp"
+      crossorigin="anonymous"
+    />
   </head>
 
   <body class="app-grid">
     <header class="app-grid__header">
-      <h1>QUIZ-APP</h1>
+      <h1 class="page-title">QUIZ-APP</h1>
     </header>
 
     <main class="app-grid__main">
-      main
-
-      <div class="flaticon-referal">
-        Icons made by
-        <a href="https://www.freepik.com" title="Freepik">Freepik</a> from
-        <a href="https://www.flaticon.com/" title="Flaticon"
-          >www.flaticon.com</a
-        >
-      </div>
+      <section class="quiz-card">
+        <a class="fas fa-bookmark quiz-card__bookmark" href=""></a>
```

Comment:
Why not use a `span`-element instead of an anchor? Anchors are typically used for linking.
```suggestion
        <span class="fas fa-bookmark quiz-card__bookmark"></span>
```